### DISCUSSION TITLE
Trouble converting CP2K data to DeepMD input file

### DISCUSSION THREAD

**Discussion by gwenwhite** [deepmodeling/dpdata#748] [Q&A] [answered]
Hello, I received the following error when converting the CP2K AIMD output file to a DeepMd input file. I am using dpdata version 0.2.21.

**My code:**
```
import dpdata

cp2kmd_dir = "cp2k-data"
cp2kmd_output_name = "cp2k-test.log"
dp = dpdata.LabeledSystem(cp2kmd_dir, cp2k_output_name=cp2kmd_output_name, fmt="cp2k/aimd_output", restart=False)
print(dp)
```
**Error:**
```
PendingDeprecationWarning: 
Hi, you got an error from dpdata,
please check if your cp2k files include full information,
otherwise its version is not supported by dpdata.
Try use dpdata plugin from cp2kdata package,
for details, please refer to
https://robinzyb.github.io/cp2kdata/
```

I tried running `pip install cp2kdata` as suggest in the link in the error. But no fix. 
Let me know if you need more information!

[cp2k-test.log](https://github.com/user-attachments/files/17592578/cp2k-test.log)
[input.txt](https://github.com/user-attachments/files/17592613/input.txt)

**Comment by robinzyb** [comment]
hello, any error message from cp2kdata ? what command you used for cp2kdata?

**Reply by gwenwhite** [reply]
`---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[3], line 1
----> 1 from cp2kdata import Cp2kOutput
      2 cp2k_output_file = "cp2k-test.log"
      3 cp2koutput = Cp2kOutput(cp2k_output_file, path_prefix=".")

ImportError: cannot import name 'Cp2kOutput' from 'cp2kdata' (unknown location)`

**Reply by robinzyb** [reply]
this seems that the package cannot found by python. I am not sure if the package is installed correctly. In principle, the pip is supposed to be located in the same environment used by the python.

**Reply by gwenwhite** [reply, marked as answer]
Decided to just start from scratch. Created a new environment, ran `pip install cp2kdata`, and things seem to be working now. I think I may installed both cp2k and dpdata or some variations of those. 
For reference here is the working code below, Thanks!!:

```
cp2k_output_file = "cp2k-test.log"
cp2koutput = Cp2kOutput(cp2k_output_file, path_prefix="cp2k-data")
print(cp2koutput)

Cp2k Output Summary

--------------------------------------

Cp2k Version       : 2023.1

Run Type           : MD

Atom Numbers       : 200

Frame Numbers      : 101

Force in Output    : Yes

Stress in Output   : Yes

Element List       : Na   Cl   

Element Numb       : 100  100  
--------------------------------------
```

**Reply by robinzyb** [reply]
glad you have made it !
